### ISSUE TITLE
Buffer overflow on long reads.

### ISSUE THREAD

**Issue by jkbonfield** [divonlan/genozip#13]
On a pacbio CLR long-read data set downloaded from ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/working/20131209_na12878_pacbio/si/NA12878.pacbio.bwa-sw.20140202.bam I get this error:

```
$ ~/lustre/genozip/genozip -@12 -f -e hs37d5.ref.genozip NA12878.pacbio.bwa-sw.20140202.bam
genozip NA12878.pacbio.bwa-sw.20140202.bam : 3% (39 minutes 29 seconds)*** buffer overflow detected ***: /nfs/users/nfs_j/jkb/lustre/genozip/genozip terminated
Aborted (core dumped)
```

This was built from the master branch, with `git describe --tags` claiming genozip-12.0.33-4-g308a84dd.  The OS is Ubuntu 18.04.5 LTS (Bionic).  I can't recall how I built it, but most likely it'd have just been "make" so using the system gcc with whatever optimisation options are listed in the Makefile.

**Comment by divonlan** [comment]
Hi James, thanks again for reporting this and thanks for your patience. This issue has now been fixed, and released in version 12.0.34.

**Comment by jkbonfield** [comment]
Thanks.
